jsx/Filter.js

```diff
@@ -74,6 +74,7 @@ function Filter(props) {
             element = (
               <SelectElement
                 options={filter.options}
+                sortByValue={filter.sortByValue ?? true}
```

Comment:
This should go into Filter.defaultProps instead of coalescing here for consistency

Reply:
I don't think that will work.
DefaultProps only work when the props is not set, but here it will be set to filter.sortByValue, which is undefined and therefore false by default.

Reply:
Wait I'm wrong I think it should work :)

Reply:
Fixed.

Reply:
Did you forget to commit/push it? The coalesce is gone but I don't see the prop defined or the default in the proptypes.

Reply:
It's already there : https://github.com/aces/Loris/blob/main/jsx/Form.js#LL680C3-L680C21